src/testUtils.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-imports */
+import React from 'react';
+import {render, RenderOptions} from '@testing-library/react-native';
+import {ApolloProvider, ApolloClient, InMemoryCache, HttpLink} from '@apollo/client';
+import fetch from 'cross-fetch';
+
+// import {LitentryApiClientProvider} from 'context/LitentryApiContext';
+import ThemeProvider from 'context/ThemeContext';
+
+export const client = new ApolloClient({
+  cache: new InMemoryCache(),
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why do we need the @ts-ignore here?

Reply:
`HttpLink` expects a `fetch` with a different interface, we are using `cross-fetch` which has a simpler interface (less config options)

Reply:
I don't see any TS errors in my machine 🤔 .
But it's fine.

Reply:
Actually that was for `node-fetch`, another library recommended to do the same but I got some TS issues with the dependencies of that library on jest and changed it. We don't need to ignore anything here now